gems/spree/OSVDB-73751.yml

```diff
@@ -0,0 +1,11 @@
+---
+gem: spree
+osvdb: 73751
```

Comment:
Yup, as soon as OSVDB is back to working order, I will go back and update all of this. Main thing is to get them added now, and the URL provides the real description.